debezium-connector-mysql/pom.xml

```diff
@@ -115,6 +115,12 @@
             <artifactId>kafka_${version.kafka.scala}</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>io.debezium</groupId>
+            <artifactId>debezium-storage-kafka</artifactId>
+            <version>2.0.0-SNAPSHOT</version>
```

Comment:
The module definition should go to `debezium-bom/pom.xml` so version is maintained there - see `Debezium artifacts`. Also please add the dependency right below `debezium-core`